dsgrid/registry/registry_manager_base.py

```diff
@@ -225,16 +227,20 @@ def _check_update(self, config, config_id, version):
         if version != cur_version:
             raise DSGInvalidParameter(f"version={version} is not current. Current={cur_version}")
 
-    def _update_config(self, config, submitter, update_type, log_message):
-        config_id = config.config_id
-        registry_config = self.get_registry_config(config_id)
-
+    @staticmethod
+    def get_next_version(version: VersionInfo, update_type: VersionUpdateType):
```

Comment:
I thought we got rid of VersionInfo? Do we just use it for a moment to calculate the new version string?

Reply:
We no longer use it in the Pydandic serialization/deserialization, but we still use it in the code.